include/swift/Runtime/Concurrency.h

```diff
@@ -900,6 +918,20 @@ SerialExecutorRef swift_task_getCurrentExecutor(void);
 SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
 SerialExecutorRef swift_task_getMainExecutor(void);
 
+/// Return the generic (default global concurrent) executor reference.
+SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
+SerialExecutorRef swift_task_getGenericExecutor(void);
+
+/// Return the "undefined" task executor reference.
+///
+/// It can be used used to store a record indicating that
+/// there is no task executor preference in the current scope.
+SWIFT_EXPORT_FROM(swift_Concurrency) SWIFT_CC(swift)
+TaskExecutorRef swift_task_getUndefinedTaskExecutor(void);
```

Comment:
Similarly, I don't think we need a function for this, because we really want this to be guaranteed to be the all-zeroes value.

Reply:
Okey gotcha so I'll just bitcast zeroes and avoid the runtime call. Thanks!

Reply:
Done, and removed the runtime func -- explained more here: https://github.com/apple/swift/pull/68793#discussion_r1398654044